applications/GeoMechanicsApplication/custom_python/add_custom_strategies_to_python.h

```diff
@@ -28,7 +28,7 @@ namespace Kratos
 namespace Python
```

Comment:
Let's simplify the nested namespaces:
```suggestion
namespace Kratos::Python
```

Reply:
Done